run_pacta_scenario_preparation.R

```diff
@@ -1,4 +1,4 @@
-library(tidyverse)
+library(dplyr)
```

Comment:
Generally prefer function namespacing `foo::bar()` over `library(foo)`. It doesn't affect/override the search path

I know this is just a sample script for the moment, so non-blocking.

Reply:
Yup understood! I agree, and will implement like that in the actual script